packages/core/services/maps-api-loader/lazy-maps-api-loader.ts

```diff
@@ -84,8 +84,9 @@ export class LazyMapsAPILoader extends MapsAPILoader {
   protected _config: LazyMapsAPILoaderConfigLiteral;
   protected _windowRef: WindowRef;
   protected _documentRef: DocumentRef;
+  protected readonly _SCRIPT_ID: string = 'agmGoogleMapsApiScript';
 
-  constructor(@Inject(LAZY_MAPS_API_CONFIG) config: any, w: WindowRef, d: DocumentRef) {
+  constructor(@Optional() @Inject(LAZY_MAPS_API_CONFIG) config: any, w: WindowRef, d: DocumentRef) {
```

Comment:
Just to be sure we're on the same page, the idea is that you don't have to provide the token at all if you're including the `script` tag on your own or expecting a different lib to do it, right?

Reply:
good catch! No, this was for "quick testing" reasons 😄  I will remove it because it's always provided: https://github.com/SebastianM/angular-google-maps/blob/master/packages/core/core.module.ts#L41

Reply:
Shouldn't the parameter `config` also match the token type? I mean: `LazyMapsAPILoaderConfigLiteral` instead of `any`